### PR TITLE
172 removed add button

### PR DESCRIPTION
**Pull request by jmclain-banyan**
😆  added the remove buttons
image

### CODE REVIEW

File: client/src/components/Inventory/Inventory.tsx [hunk 1]
```diff
@@ -1846,6 +1852,15 @@ const Inventory: FC = () => {
                                     <FaPlus />
                                     Add Value
                                 </button>
+                                <button
+                                    tabIndex={-1}
+                                    onClick={(e) =>
+                                        removeItem(e, 'exteriorFinish')
```
Comment (owen-the-terrible) [inline client/src/components/Inventory/Inventory.tsx:1858]: Does this line need `designStyle`  included in that removeItem function?

File: client/src/components/Inventory/Inventory.tsx [hunk 2]
```diff
@@ -1888,6 +1903,15 @@ const Inventory: FC = () => {
...
```
Comment (owen-the-terrible) [inline client/src/components/Inventory/Inventory.tsx:1909]: Also does this one need 'usePackages' included in the removeItem function?

**Review by owen-the-terrible — CHANGES_REQUESTED** [review]
If you tested them out and they're working the right way then cool but will you give those a looksie?

File: client/src/components/Inventory/Inventory.tsx (hunk 1 above)
Comment (jmclain-banyan) [reply client/src/components/Inventory/Inventory.tsx:1858]: my bad, good catch.

File: client/src/components/Inventory/Inventory.tsx (hunk 2 above)
Comment (jmclain-banyan) [reply client/src/components/Inventory/Inventory.tsx:1909]: another one, you're right

**Review by owen-the-terrible — APPROVED** [review]
Thanks bruh

**Pull request merged**